templates/index.html

```diff
@@ -181,10 +173,10 @@ <h3>Automate app testing at scale</h3>
   </div>
   <div class="row">
     <div class="col-7">
-      <p >Do you have other use cases in mind for Android in the cloud with Anbox Cloud?</p>
+      <p>Do you have other use cases in mind for Android in the cloud with Anbox Cloud?</p>
     </div>
     <div class="col-5">
-      <a class="p-button--positive" href="/contact-us">Tell us more</a>
+      <a class="p-button js-invoke-modal" href="/contact-us">Tell us more</a>
```

Comment:
Shouldn't this should be a more prominent positive button?

Reply:
We cannot have more than one positive button per page.

Reply:
I think there is a misunderstanding here. Only one positive CTA should be visible per screen, but the three buttons are far from each other, so it should be fine to have them all positive.